docs/guides/jupyter.rst

```diff
@@ -0,0 +1,293 @@
+How to use Jupyter notebooks in Sphinx
+======================================
+
+By default, `Sphinx only supports reStructuredText source files for
+documentation <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix>`_.
+Luckily, there are a few extensions to also allow Jupyter notebooks as source files.
+
+.. _ipynb-notebooks-sphinx:
+
+Including ``.ipynb`` notebooks in Sphinx documentation
+------------------------------------------------------
+
+There are two main extensions that add support for ``.ipynb`` as source files in Sphinx:
+nbsphinx_ and MyST-NB_. Even though they have some differences,
+their basic mode of operation is very similar. However, they are mutually exclusive,
+so you have to choose one of them.
```

Comment:
This feels scary. Are we going to tell users which one to use?! How do they work? 

We just go right into step-by-step content after this introduction, which feels weird.

Reply:
It _is_ scary 😄 now seriously, I was trying to avoid getting into too much technical detail, but you're right we should distill that information in a less intimidating way. Rewriting.